components/Caretaker.js

```diff
@@ -77,11 +76,13 @@ const styles = StyleSheet.create({
     width: "auto", 
     alignSelf: "flex-end",
     textAlign: "center",
-    margin: 20,
+    marginTop: 50,
+    marginRight: 30,
```

Comment:
You said that this still isn't dynamic across all platforms? This might be something that we do need to change. I'll make an issue for it.